File: examples/hello_world.c
#include "stdio.h"
#include "math.h"
void main(int argc, char* argv[]) {
    printf("Hello, world!\n");
    int age = 27;
    if (age > 18) {
        printf("You are an adult!");
    } else {
        if (age > 12 && age < 18) {
            printf("You are a teenager!");
        } else {
            printf("You are younger than 12!");
        }
    }
}

File: examples/hello_world.py
import stdio, math

def main(argc: int, argv: list[str]):
    printf("Hello, world!\n")
    age: int = 27
    if age > 18:
        printf("You are an adult!")
    elif age > 12 and age < 18:
        printf("You are a teenager!")
    else:
        printf("You are younger than 12!")